Add new ability mapping for Spiked Whip

tr1age opened the pull request:
Fixes Whirlwind attack for whip.

tr1age committed:
Add new ability mapping for Spiked Whip

Fixes Whirlwind attack for whip.

Lichtblitz commented on src/ClassicUO.Client/Game/Data/Ability.cs:
```diff
@@ -306,6 +306,7 @@ internal static class AbilityData
             { 0x48D0, new (0x48D0, Ability.Feint, Ability.DoubleStrike) },             // Gargish Daisho
             { 0xA289, new (0xA289, Ability.ConcussionBlow, Ability.WhirlwindAttack) }, // Barbed Whip
             { 0xA28A, new (0xA28A, Ability.ArmorPierce, Ability.WhirlwindAttack) },    // Spiked Whip
+            { 0xA292, new (0xA292, Ability.ArmorPierce, Ability.WhirlwindAttack) },    // Spiked Whip
```
<img width="213" height="183" alt="Image" src="https://github.com/user-attachments/assets/5abe2868-7e09-4358-8f67-9f191ac71fc0" />

There is also a Barbed and Bladed Whip at 0xA291 and 0xA293. If you know the abilities, it would be great to add those as well.

tr1age replied:
They are the same so added them as secondary IDs Although these are just FLIPPED ids. So, I don't think this actually matters. I am removing it.

tr1age committed:
Add additional ability IDs for Barbed and Bladed Whips

They are the exact same so added the second IDs

tr1age committed:
Remove duplicate ability entries for Barbed and Bladed Whips

Don't need the extras they are only the flipped ID. Once held the ID listed is used.

tr1age reviewed (commented):
This doesn't actually matter since these are JUST the flipped ID for the items.

Lichtblitz commented on src/ClassicUO.Client/Game/Data/Ability.cs:
Broadswords for example have their flipped counterparts added as well. It makes sense since it's the server, not the client that knows about flipped IDs. From the client's perspective those graphics could as well be different items.

tr1age replied:
Ahh didn't know they did it that way, Adding.

tr1age committed:
Add flipped ID entries for abilities

Lichtblitz reviewed (approved):
LGTM
